commands/scp.go

```diff
@@ -148,6 +150,11 @@ func getInfoForScpArg(hostAndPath string, hostInfoLoader HostInfoLoader) (HostIn
 	}
 
 	args := []string{}
+	port, err := hostInfo.GetSSHPort()
+	if err == nil && port > 0 {
+		args = append(args, "-P", fmt.Sprintf("%v", port))
+	}
```

Comment:
Are you sure the `-P` will work for multiple remote machines? I tried it on both OSX and Ubuntu 16.04 without any success.

```bash
$ scp -P 231 root@138.68.61.69:./README.md -P 232 root@138.68.50.80:./
ssh: connect to host 138.68.50.80 port 22: Connection refused
lost connection
ssh: connect to host 138.68.50.80 port 231: Connection refused
lost connection
```

SSH port of `138.68.61.69` is `231`, and SSH port of `138.68.50.80` is `232`. I looks like the `scp` only use one of `-P` for its port setting.